setup.py

```diff
@@ -24,7 +24,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "prometheus_client>=0.2.0,<0.5.0",
+        "prometheus_client>=0.2.0",
```

Comment:
For safety's sake can we restrict the upper limit on this requirement?
```suggestion
        "prometheus_client>=0.2.0,<1.0.0",
```